lib/controllers/agendash.js

```diff
@@ -309,7 +310,8 @@ module.exports = function (agenda, options) {
 
     const collection = agenda._collection.collection || agenda._collection;
```

Comment:
@ladislavjacho can we remove this declaration, and the commented line below? looks like this PR is not getting merged until tests pass. thanks for the PR, frustrating that this core functionality does not work :/